### PR TITLE
Create export to Terra "not started" component. #468.

### LINKED ISSUE #468: Create export to Terra "not started" component
#### Need
Once a user has selected to the "export data to Terra" export method, we display a page with an export options form and a button to initiate the export. See [mocks](https://www.figma.com/file/i4uZsYkKAML9ivi2TIo6ug/explore-dashboard?node-id=1012%3A19397). We need a single component to display the core of this export page: the introductory text, export form and request link button.

#### Approach
* For now, the export options form is out of scope for this ticket. 
* For now, "hard code" this component. Converting this component to be configurable (with a view model builder etc) will be handled in a separate ticket.
* Create an `ExportToTerraNotStarted` component in `app/components/ExportToTerra`.
* The component takes a single prop: `run`, which is a function.
* Re-use existing `BatchNormalizationWarning` component.
* Attach a click handler to the "Request Link" button that invokes the `run` function passed in a prop. 

#### Definition of Done
* Component only displays the batch normalization warning and the "Request Link" button.
* On click of the "Request Link" button, `run` is invoked.

### PR DESCRIPTION
**Pull request by frano-m**
### Ticket
Closes #468.

### Reviewers
@MillenniumFalconMechanic.

### Changes
- Added Terra "not started" component and SB.

### CODE REVIEW

File: explorer/app/components/Export/components/ExportToTerra/components/ExportToTerraNotStarted/exportToTerraNotStarted.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+interface Props {
+  run: RunFn;
+}
+
+export const ExportToTerraNotStarted = ({ run }: Props): JSX.Element => {
+  return (
+    <RoundedPaper>
+      <Section>
+        <SectionContent gap={2}>
+          <SectionTitle title="Select File Types" />
```
Comment (MillenniumFalconMechanic) [inline explorer/app/components/Export/components/ExportToTerra/components/ExportToTerraNotStarted/exportToTerraNotStarted.tsx:26]: Is it possible to just have the batch normalization warning and request link button as the section content here?

**Review by MillenniumFalconMechanic — APPROVED** [review]
